Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:10.12.0-alpine
+LABEL maintainer="ESI Middleware Team <bl-uits-esi-sit@exchange.uits.iu.edu>"
+
+WORKDIR /usr/src/app
+ENTRYPOINT ["/usr/src/app/docker-entrypoint.sh"]
+CMD ["all"]
+
+RUN apk add --no-cache git
```

Comment:
It took me a minute to figure it out, but I'm guessing this `RUN` is in a separate layer because the `npm install` needs it to get your fork of Caporal? If so, you might put a comment here explaining why it's separate.

Reply:
It is required for the `npm install`, but it is separate from the `npm install` line because I wouldn't want to have to reinstall this again every time the `package.json` file changes.

Reply:
Gotcha